doc/chomp_interface_tutorial.rst

```diff
@@ -12,33 +12,40 @@ Once you have this package simply run::
 
-Assumptions:
-------------
+Applying CHOMP with your own robot
+-------------------------------
 
- 1. You have the latest version of moveit installed. On ROS kinetic you may need to build it from source.
- 2. You have a moveit configuration package for your robot already. For example, if you have a Kinova Jaco arm, it's probably called "jaco_moveit_config". This is typically built using the Moveit Setup Assistant.
- 3. Lets assume that you are using the **jaco** manipulator. And hence, the moveit config package is *jaco_moveit_config*.
+Assumptions
+============
 
+ 1. You have the latest version of moveit installed. On ROS kinetic you may need to build it from source.
+ 2. You have a moveit configuration package for your robot already. For example, if you have a `Fetch robot <http://fetchrobotics.com/platforms-research-development>`_, it's probably called `fetch_moveit_config <https://github.com/fetchrobotics/fetch_ros/tree/indigo-devel/fetch_moveit_config>`_. This is typically built using the Moveit Setup Assistant.
 
-Using CHOMP with your own robot
--------------------------------
+Step to apply CHOMP
+====================
 
-1. Simply download `chomp_planning_pipeline.launch.xml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/launch/chomp_planning_pipeline.launch.xml>`_ file into the launch directory of your moveit config package. So into the *jaco_moveit_config/launch* directory.
-2. Adjust the line <rosparam command="load" file="$(find moveit_resources)/fanuc_moveit_config/config/chomp_planning.yaml" /> to <rosparam command="load" file="$(find jaco_moveit_config)/config/chomp_planning.yaml" />
-3. Download `chomp_planning.yaml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/config/chomp_planning.yaml>`_ file into the config directory of your moveit config package. So into the *jaco_moveit_config/config* directory.
-4. Copy the *demo.launch* file to *demo_chomp.launch*. Note that this file is also in the launch directory of the *jaco_moveit_config* package.
+1. Simply download `chomp_planning_pipeline.launch.xml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/launch/chomp_planning_pipeline.launch.xml>`_ file into the launch directory of your moveit config package. So into the *fetch_moveit_config/launch* directory.
+2. Adjust the line `<rosparam command="load" file="$(find moveit_resources)/fanuc_moveit_config/config/chomp_planning.yaml" />` to `<rosparam command="load" file="$(find fetch_moveit_config)/config/chomp_planning.yaml" />`
+3. Download `chomp_planning.yaml <https://github.com/ros-planning/moveit_resources/blob/master/fanuc_moveit_config/config/chomp_planning.yaml>`_ file into the config directory of your moveit config package. So into the *fetch_moveit_config/config* directory.
+4. Copy the *demo.launch* file to *demo_chomp.launch*. Note that this file is also in the launch directory of the *fetch_moveit_config* package.
```

Comment:
it would be nice to soon just have all these configuration modifications in the setup assistant templates